Sprint-2/errors/0.js

```diff
@@ -1,9 +1,17 @@
 // Predict and explain first...
 
-// call the function capitalise with a string input
+// call the function capitalize with a string input
 // interpret the error message and figure out why an error is occurring
 
 function capitalise(str) {
-  let str = `${str[0].toUpperCase()}${str.slice(1)}`;
+   str = `${str[0].toUpperCase()}${str.slice(1)}`;
+  return str;
+}
+//SyntaxError: Identifier 
+// First, as the function parameter ==> function capitalise(str){}
+// Second, as a let variable inside the function, let str
+//to fix this issue :
+function capitalise(str) {
+   str = `${str[0].toUpperCase()}${str.slice(1)}`;
   return str;
```

Comment:
Nice spot, how would you describe this error in the function if you were talking to a friend?

Reply:
This is  a small issue here because the name str is being used in two different ways that might be conflicting. First, it's used as a parameter in the function definition, and then inside the function, it's being treated as a local variable.